### ISSUE TITLE
Log utebnti che accettano i cookie

### ISSUE THREAD

**Issue by paologalli** [overclokk/italy-cookie-choices#75]
ciao, Il Garante afferma che è necessario tenere il log degli utenti che accettano i cookie. il tuo plugin potrebbe tenere questo logo? al momento non credo lo faccia

**Comment by acardinale** [comment]
A me sembra una cosa assurda. I siti (tutti) devono quindi profilare gli utenti?

**Comment by paologalli** [comment]
no, non profilare. il garante però vuole si tenga traccia del loro consenso

**Comment by acardinale** [comment]
Secondo me è la stessa cosa, come "tenere traccia di chi ha aggiunto un prodotto nel carrello", tenere traccia di chi ha cliccato un banner. Non capisco cosa cambia.
In questo modo io potrei fare una homepage diversa in base al giorno dell'accettazione (giorno pari faccio vedere una home, giorno dispari ne faccio vedere un'altra).

**Comment by ghost** [comment]
Ciao, scusate l'intromissione.

Io credo cambi più che altro la (supposta) finalità. 
Capisco che è ridicolo legare una norma a un'intenzione, ma più cerco di capirla più mi sembra che la legge, per assurdo che sia, ci chieda proprio quello di cui parlava paolo.
Se mi si obbliga a registrare un ip e a correlarlo al valore y del cookie DisplayCookieConsent poi non mi si può accusare di averlo indebitamente registrato.

**Comment by acardinale** [comment]
@ChiaraSO non intendo che è "indebito", ma che per me significa profilare e per il garante profilare significa fare la notifica. Poi non so se questo, in particolare è escluso dalla notifica.

**Comment by andreapernici** [comment]
Profilare significa che poi gli provi a vendere qualcosa.

Quindi tenere i log a fini tecnici non è profilazione.

**Comment by overclokk** [comment]
Il problema è come faccio ad essere sicuro che quell'IP, quell'user agent ecc siano realmente di quella persona?

**Comment by andreapernici** [comment]
Il problema è in capo a chi denuncia poi di dimostrare che non lo è.

**Comment by ghost** [comment]
Forse però il problema della persona non rientra tra gli interessi del Garante (proprio perché non è risolvibile). Penso sia più verosimile che, a fronte di una contestazione, si possa fare un confronto tra gli ip registrati nei log "generali" e quelli registrati nei log di consenso, che a rigor di logica dovrebbero combaciare per chi lo ha fornito.

**Comment by acardinale** [comment]
@andreapernici questa definizione di wikipedia la reputi corretta?
_Per profilazione dell'utente si intende correntemente l'insieme di attività di raccolta ed elaborazione dei dati inerenti agli utenti di servizi (pubblici o privati, richiesti o forzosi) per suddividere l'utenza in gruppi di comportamento_

In questo caso "suddivido" tra chi accetta e chi no.

Ricordiamoci che non parliamo di tecnico ma di "avvocatese" in cui anche una virgola non troppo evidente può far dire che era un "punto" e che dunque... :)

Dunque è certo che occorrerà salvare questi "log"?

**Comment by andreapernici** [comment]
Non lo so, ma in teoria che io sappia il nodo della questione è il cookie di profilazione a fini pubblicitari.

Se così non fosse sarebbe profilazione anche tenere i log del server...il che non ha senso.

**Comment by andreapernici** [comment]
Leggi questo
http://www.allaboutcookies.org/cookies/cookie-profiling.html

**Comment by ghost** [comment]
Ciao, 
rispolvero la discussione perché oggi ho ricevuto una telefonata da parte di una rappresentate dell'ufficio del Garante.
Mi ha dedicato un bel po' di tempo e mi ha risposto ad alcune domande poste un po' di tempo fa via mail, tra le quali c'era appunto anche quella relativa al consenso.

Ve la riassumo, anche se si tratta di una comunicazione informale fatta via telefono.

1) La necessità di registrare il consenso discende dall'Art. 23 del Codice, che si applica a tutti i consensi espressi. Non è una condizione prescindibile.
2) Il cookie tecnico di consenso deve essere salvato sia sul dispositivo dell'utente che sul server della prima parte. (Immagino sia sottinteso le informazioni contenute nel cookie).
3) Ho fatto presente che le informazioni presenti sul cookie non sono affatto in grado di identificare l'utente ma solo di verificare se il cookie è stato creato, quando e se il valore y conferma che ha accettato.
4) In questo caso, la persona con cui ho parlato mi ha confermato che è necessario arricchire la nostra "copia del cookie" con altre informazioni, in un log contenente tutti gli elementi dai quali sia possibile desumere che il consenso è stato registrato. L'esempio che abbiamo fatto è quello dell'indirizzo ip.
5) Ho fatto presente che l'indirizzo IP non è assolutamente sufficiente a identificare univocamente un utente. La persona al telefono mi ha spiegato che sono consapevoli delle difficoltà del caso, ma ha anche detto che non possono darmi una lista precisa dei dati da raccogliere nel log.
6) Stanno riflettendo sulle modalità per rendere il consenso in qualche modo certificabile, ma non esistono ancora, allo stato delle cose, linee guida precise.

In sostanza non ci sono, al momento, liste ufficiali dei dati da raccogliere nei registri di consenso.

Ha anche spiegato che a recepire eventuali segnalazioni sono persone dotate di buon senso e di esprienza, rassicurandomi sul fatto che sarà cura del garante procedere cum grano salis a fronte di segnalazioni in malafede.
Di questo sinceramente non dubito. 
Restano di fatto l'impossibilità di creare un registro a prova di bomba e la necessità di creare un registro.

**Comment by overclokk** [comment]
Va beh, che dire, questi proprio stanno degenerando, secondo me non si rendono conto.

Come si fa a contattare queste persone?

**Comment by ghost** [comment]
Via mail: urp(at)gpdp.it
E' il metodo che ho usato io. Ho scritto loro nel periodo di massima "crisi" quindi ci hanno messo un bel po' a rispondermi, con una richiesta di contatto telefonico. Insomma, tempi di attesa lunghini ma quando ti chiamano ti ascoltano e non mettono fretta. 

Telefono: (+39) 06.69677.2917
lunedì - venerdi ore 10-13

Per una comunicazione di carattere più ufficiale prova la PEC: urp(at)pec.gpdp.it.

**Comment by overclokk** [comment]
Grazie @ChiaraSO grazie delle info.

**Comment by ghost** [comment]
È stato poi implementato un log dei consensi?

**Comment by overclokk** [comment]
@lupett cosa intendi?

**Comment by ghost** [comment]
@overclokk un log degli utenti che accettano l'informativa breve. Come si parlava in questo ticket. :)

**Comment by overclokk** [comment]
@lupett no, non è stato implementato nulla a riguardo.

**Comment by ghost** [comment]
@overclokk grazie, lo implementerete con qualche aggiornamento o c'è modo di crearlo tramite altro metodo?
